libraries/holochain-service-client/src/main/aidl/org/holochain/androidserviceruntime/holochain_service_client/AppAuthFfiParcel.aidl

```diff
@@ -0,0 +1,3 @@
+package org.holochain.androidserviceruntime.holochain_service_client;
```

Comment:
Parcelable AIDL types needed to be specified in their own file so they could be imported to multiple interfaces.